app/components/chat.tsx

```
@@ -153,7 +91,10 @@ export function Chat({ repo, path, setPath, graph, selectedPathId, isPathRespons
         setPaths([])
     }, [isPathResponse])
 
-    const handelSetSelectedPath = (p: PathData) => {
+    const handleSetSelectedPath = (p: PathData) => {
+        const canvas = canvasRef.current
+
+        if (!canvas) return
         setSelectedPath(prev => {
             if (prev) {
                 if (isPathResponse && paths.some((path) => [...path.nodes, ...path.links].every((e: any) => [...prev.nodes, ...prev.links].some((e: any) => e.id === e.id)))) {
```

Biome (2.4.4)

[error] Comparing to itself is potentially pointless.

(lint/suspicious/noSelfCompare)